### PR TITLE
[v2.10.0] Add docs about enable/disable functionality in clusterrepo

### PR DESCRIPTION
**Pull request by rohitsakala**
Please check the issue for more details 

### Summary 

- A new feature about enabling and disabling a Helm repository is added. Updating the docs related to it.
- Targeted for 2.10.0

### CODE REVIEW

File: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md [hunk 1]
```diff
@@ -194,6 +194,22 @@ Non-Airgap Rancher installations upon refresh will reflect any chart repository
 
 Airgap installations where Rancher is configured to use the packaged copy of Helm system charts ([`useBundledSystemChart=true`](../../../getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/install-rancher-ha.md#helm-chart-options-for-air-gap-installations)) will only refer to the [system-chart](https://github.com/rancher/system-charts) repository that comes bundled and will not be able to be refreshed or synced.
 
+### Enable/Disable Helm Chart Repositories
+
+A new feature is added to enable and disable a Helm repository. A Helm repository is enabled by default.
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md:199]:
```suggestion
Rancher v2.10.0 adds the ability to enable and disable Helm repositories. Helm repositories are enabled by default.
```

File: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md [hunk 2]
```diff
@@ -194,6 +194,22 @@ Non-Airgap Rancher installations upon refresh will reflect any chart repository
...
+
+To disable a chart repository:
+
+1. Find the repository you want to disable, and click **⋮ > Edit YAML**.
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md:203]: We should add how to navigate to the place where repositories are listed

File: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md [hunk 3]
```diff
@@ -194,6 +194,22 @@ Non-Airgap Rancher installations upon refresh will reflect any chart repository
...
+1. Set the `enabled` field under `Spec` to false.
+1. Click on **Save**.
+1. When you disable a repository, updates are disabled and new changes to the clusterRepo are not picked up.
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md:206]:
```suggestion
1. When you disable a repository, updates are disabled and new changes to the clusterRepo are not applied.
```

File: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md [hunk 4]
```diff
@@ -194,6 +194,22 @@ Non-Airgap Rancher installations upon refresh will reflect any chart repository
...
+
+To enabled a chart repository:
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md:208]:
```suggestion
To enabled a chart repository:

```

File: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md [hunk 5]
```diff
@@ -194,6 +194,22 @@ Non-Airgap Rancher installations upon refresh will reflect any chart repository
...
 Airgap installations where Rancher is configured to use the packaged copy of Helm system charts ([`useBundledSystemChart=true`](../../../getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/install-rancher-ha.md#helm-chart-options-for-air-gap-installations)) will only refer to the [system-chart](https://github.com/rancher/system-charts) repository that comes bundled and will not be able to be refreshed or synced.
 
+### Enable/Disable Helm Chart Repositories
+
+A new feature is added to enable and disable a Helm repository. A Helm repository is enabled by default.
+
+To disable a chart repository:
+
+1. Find the repository you want to disable, and click **⋮ > Edit YAML**.
+1. Set the `enabled` field under `Spec` to false.
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md:204]:
```suggestion
1. Set the **Enabled** field under **Spec** to **false**.
```

File: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md [hunk 6]
```diff
@@ -194,6 +194,22 @@ Non-Airgap Rancher installations upon refresh will reflect any chart repository
...
+1. Click on **Save**.
+1. When you disable a repository, updates are disabled and new changes to the clusterRepo are not picked up.
+
+To enabled a chart repository:
+1. Find the repository you want to disable, and click **⋮ > Edit YAML**.
+1. Set the `enabled` field under `Spec` to true.
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md:210]:
```suggestion
1. Set the **Enabled** field under **Spec** to **true**.
```

File: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md [hunk 7]
```diff
@@ -194,6 +194,22 @@ Non-Airgap Rancher installations upon refresh will reflect any chart repository
...
+1. Click on **Save**.
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md:211]:
```suggestion
1. Click **Save**.
```

File: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md [hunk 8]
```diff
@@ -194,6 +194,22 @@ Non-Airgap Rancher installations upon refresh will reflect any chart repository
...
 
+### Enable/Disable Helm Chart Repositories
+
+A new feature is added to enable and disable a Helm repository. A Helm repository is enabled by default.
+
+To disable a chart repository:
+
+1. Find the repository you want to disable, and click **⋮ > Edit YAML**.
+1. Set the `enabled` field under `Spec` to false.
+1. Click on **Save**.
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md:205]:
```suggestion
1. Click **Save**.
```

**Review by martyav — CHANGES_REQUESTED** [review]

**Review by martyav — APPROVED** [review]

**Review by martyav — APPROVED** [review]

**Review by btat — APPROVED** [review]

**Comment by btat** [comment]
Updated base branch since we have a v2.10 release branch now.

Rebased to pick up update incorporating 2.10 directory. Update has been applied to both en/version-2.10 and zh/version-2.10 folders.

@LucasSaintarbor or @sunilarjun could you please take a second look just to make sure the porting was done correctly.

**Review by sunilarjun — APPROVED** [review]
LGTM

**Pull request merged**